packages/commerce/backend/src/open-payments/service.ts

```diff
@@ -0,0 +1,202 @@
+import { TokenCache } from '@/cache/token'
+import { Env } from '@/config/env'
+import { InternalServerError } from '@/errors'
+import { Order } from '@/order/model'
+import { extractUuidFromUrl } from '@/shared/utils'
+import {
+  AuthenticatedClient,
+  Grant,
+  GrantRequest,
+  PendingGrant,
+  Quote,
+  isPendingGrant
+} from '@interledger/open-payments'
+import { randomUUID } from 'crypto'
+import { Logger } from 'winston'
+
+interface PreparePaymentParams {
+  order: Order
+  paymentPointer: string
```

Comment:
rename to `paymentPointerUrl` maybe, so it will be less confusing where is a mix of payment pointer objects and urls